collector/lib/Logging.cpp

```diff
@@ -100,6 +100,56 @@ bool ParseLogLevelName(std::string name, LogLevel* level) {
   return true;
 }
 
+void InspectorLogCallback(std::string&& msg, sinsp_logger::severity severity) {
+  auto collector_severity = logging::InspectorSeverityToLogLevel(severity);
+
+  if (!collector_severity) {
+    return;
+  }
+
+  collector::logging::LogMessage(__FILE__, __LINE__, false, *collector_severity) << msg;
```

Comment:
Couple notes on this line. We might want a new constructor for `LogMessage` that doesn't log file and line, since all messages from the inspector end up showing up as coming from this line. We might also want to add a prefix to signify the message is coming from the inspector, something like this:
```cpp
collector::logging::LogMessage(__FILE__, __LINE__, false, *collector_severity) << "inspector: " << msg;
```
Didn't do any of these because this initial change is good enough IMO, but would love to hear thoughts from other people.

Reply:
it would be extremely helpful if upstream provided the file and line number down to this level -- perhaps we could try to implement it there?

Reply:
We could, but for now we have other stuff for upstreaming that is further up in the pipeline. I might try to take a stab at it when I get a minute.